support/Pipelines/Bbh/Inspiral.py

```diff
@@ -20,6 +20,25 @@
 
 INSPIRAL_INPUT_FILE_TEMPLATE = Path(__file__).parent / "Inspiral.yaml"
 
+# Resolution levels defined in terms of p-refinement
+# To be replaced once AMR is used.
+INSPIRAL_LEVS = [
+    {
+        "Lev number": lev_number,
+        "Label": f"Lev{lev_number}",
+        "refinement level": 1,
+        "polynomial order": 7 + lev_number,
+    }
+    for lev_number in range(-2, 11)
+]
+
+# If no resolution is specified, use a default level
+DEFAULT_LEV = None
```

Comment:
Maybe we don't actually need a default lev. I suggest we try without a default for now and just require people to either specify lev or both L and P.